backend/tracim_backend/tests/__init__.py

```diff
@@ -120,15 +121,16 @@ def setUp(self) -> None:
         # TODO - G.M - 2019-03-19 - Replace this code by something better, see
         # https://github.com/algoo/hapic/issues/144
         hapic._controllers = []
-        self.connect_database(create_tables=True)
         self.app_config = CFG(self.settings)  # type: CFG
+        self.override_app_config()
```

Comment:
I think this is not the way to do this properly, no?

Reply:
you're right, i just refactored it.